service-registry-client/src/main/java/no/difi/meldingsutveksling/serviceregistry/externalmodel/ServiceRecord.java

```diff
@@ -40,6 +42,10 @@ public ServiceRecord() {
         this.returnAddress = PostAddress.EMPTY;
     }
 
+    public static Predicate<? super ServiceRecord> isServiceIdentifier(String identifier) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove usage of generic wildcard type. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1452)